matlab/.renku/renku.ini

```diff
@@ -0,0 +1,5 @@
+[interactive]
+default_url = /vnc
+cpu_request = 1
+mem_request = 4G
+disk_request = 2G
```

Comment:
these seem pretty specific - are you sure you want to set them as defaults?

Reply:
Yeah Matlab dies if you don't set these high enough, so I think it's best to avoid people accidentally going too low

Reply:
cool, all good just wanted to make sure it was not a leftover